aiosmtpd/docs/NEWS.rst

```diff
@@ -20,6 +20,8 @@ Fixed/Improved
 * No more failures/DeprecationWarnings for Python 3.8 (fixes #167)
 * Faster ``_handle_client()`` processing
 * Faster method access for ``smtp_*``, ``handle_*``, and ``auth_*`` hooks
+* Fix connection lost will print errors ConnectionResetError. (fixes #162)
+* Make sure that handle_exception() do not called when the client disconnected. (fixes #127)
```

Comment:
Probably typo. Should have said:

`Make sure that handle_exception() do not get called when the client disconnected.`

Reply:
Changed